recipes/sunpy/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set version = "0.6.4" %}
+
+package:
+  name: sunpy
+  version: {{version}}
+
+source:
+  fn: sunpy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/s/sunpy/sunpy-0.6.4.tar.gz 
+  md5: 45da6a4328fc5c85c717686f8aae2d23
+
+build:
+  number: 0
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+  skip: True  # [py>27]
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >1.7.1
+    - astropy >=1.0
+    - scipy
+    - matplotlib
+    - pandas
+    - mock
+    - mkl 11.3.1  # [win]
```

Comment:
That is OK BTW. Just open an issue in the feedstock after the merge so we can keep track. (Maybe add a comment here too xref to the bug.)